packages/gatsby-plugin-gatsby-cloud/src/__tests__/mocks/handlers.js

```diff
@@ -51,4 +51,12 @@ export const handlers = [
   rest.post(`http://test.com/events`, async (req, res, ctx) =>
     res(ctx.json({ message: `success` }))
   ),
+  rest.get(
+    `https://build-123-changed.gtsb.io/page-data/index/page-data.json`,
+    async (req, res, ctx) => res(ctx.text("abcdefg" + Date.now().toString()))
```

Comment:
nit: change "abcdefg" to something descriptive to make logging better. Like "This response will always change" and "This response will always be the same"